src/webgpu/shader/validation/wgsl/runtime-array-not-block-decorated.fail.wgsl

```diff
@@ -0,0 +1,10 @@
+# v-0031: struct 'S' has runtime-sized member but it's storage class is not 'storage'.
```

Comment:
There is no `var` in this example, so nothing with a storage class, is the failure due to the missing `block` decoration?